webapp/channels/src/components/channel_settings_modal/channel_settings_modal.tsx

```diff
@@ -163,7 +169,7 @@ function ChannelSettingsModal({channelId, isOpen, onExited, focusOriginElement}:
             uiName: formatMessage({id: 'channel_settings.tab.configuration', defaultMessage: 'Configuration'}),
             icon: 'icon icon-cog-outline',
             iconTitle: formatMessage({id: 'generic_icons.settings', defaultMessage: 'Settings Icon'}),
-            display: false, // this tab is not implemented yet so hiding it
+            display: shouldShowConfigurationTab,
```

Comment:
0/5 would be nice to have tests around this. Not sure how testable it is right now.

Reply:
Done.